src/maps/filters.ts

```diff
@@ -235,6 +235,10 @@ export const mapDeploymentFilter: MapFunction<DeploymentFilter, DeploymentFilter
     },
     is_schedule_active: toEquals(source.isScheduleActive),
     work_queue_name: toAny(source.workQueueName),
+    flow_or_deployment_name: toLike(source.flowOrDeploymentNameLike),
```

Comment:
oh nice, does this mean we can fuzzy search across both?!

Reply:
yes!